test_that("exists", {
  expect_no_error(main())
})
